### ISSUE TITLE
Email accounts are visible for all users

### ISSUE THREAD

**Issue by rchovan** [pierre-alain-b/rainloop-nextcloud#149]
Hi, 
I'm trying Rainloop on my own NextCloud installation, but when I've added to my account 2 mailboxes, all of them are visible in other NextCloud account too (even for admin user). 

Can be this behavior disabled, or it is bug ? 

My account: 
![image](https://user-images.githubusercontent.com/6713920/77777396-cd75d400-704f-11ea-885a-c8855c51a48d.png)

Admin account:
![image](https://user-images.githubusercontent.com/6713920/77777422-d5357880-704f-11ea-925d-b29c968d4722.png)

Another user account:
![image](https://user-images.githubusercontent.com/6713920/77777440-db2b5980-704f-11ea-917c-2a7fcb5465a1.png)

**Comment by pierre-alain-b** [comment]
There is something i do not understand. Do the accounts open automatically in each account? When I open Rainloop in a new account, it asks for a login/password and does not automatically open an already existing account...
How is authentication set in the Rainloop settings ?
![image](https://user-images.githubusercontent.com/386857/77795213-7a127e80-706d-11ea-9233-b729742e6039.png)

**Comment by rchovan** [comment]
Hi, 
I wan to have one mailbox shared for multiple users. But I don't expect, that another mailbox will be shared too. I prefer to open mail account without login, so I've changed sign_me_auto = "DefaultOn" in application.ini.  Rainloop settings is as on your image : 
- Let the user define its own email/password credentials only

1.  I have added my 2 mailboxes, one shared and one private to my account
1.  Then I've logged in different browser with different NC account, and added shared mailbox. After login, user has access to my private mailbox too. 
1. Then I've tried same steps with admin account, and result is same.
